lib/sidekiq-status/client_middleware.rb

```diff
@@ -35,5 +37,13 @@ def display_args(msg, queue)
       # For Sidekiq ~> 2.7
       return msg['args'].to_a.empty? ? nil : msg['args'].to_json
     end
+
+    def record_initial_status?(klass)
+      klass = Object.const_get(klass) if klass.is_a?(String)
```

Comment:
according to the YARD doc for the `#call` method, this should always be a Class, but I'm not sure how solidly we can trust that...

Reply:
when I ran the specs it ended up being a String
so my trust is low

Reply:
Yeah I was confused by this but then I checked [the docs](https://github.com/mperham/sidekiq/wiki/Middleware#client-side-middleware)
> Client middleware may receive the class argument as a Class object or a String containing the name of the class.